README.md

```diff
@@ -206,20 +186,35 @@ Support this project by becoming a sponsor. Your logo will show up here with a l
 <a href="https://opencollective.com/verdaccio/sponsor/8/website" target="_blank"><img src="https://opencollective.com/verdaccio/sponsor/8/avatar.svg"></a>
 <a href="https://opencollective.com/verdaccio/sponsor/9/website" target="_blank"><img src="https://opencollective.com/verdaccio/sponsor/9/avatar.svg"></a>
 
-<h2 align="center"> FAQ / Contact / Troubleshoot</h2>
+## Backers
+
+Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/verdaccio#backer)]
+
+<a href="https://opencollective.com/verdaccio#backers" target="_blank"><img src="https://opencollective.com/verdaccio/backers.svg?width=890"></a>
+
+## Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="../../graphs/contributors"><img src="https://opencollective.com/verdaccio/contributors.svg?width=890&button=false" /></a>
+
+
+### FAQ / Contact / Troubleshoot
 
 If you have any issue you can try the following options, do no desist to ask or check our issues database, perhaps someone has asked already what you are looking for.
 
-* [Roadmap](https://github.com/verdaccio/verdaccio/wiki)
-* [Most common questions](https://github.com/verdaccio/verdaccio/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Aquestion%20)
-* [Reporting a bug](https://github.com/verdaccio/verdaccio/blob/master/CONTRIBUTING.md#reporting-a-bug)
+* [Blog](https://medium.com/verdaccio)
+* [Donations](https://opencollective.com/verdaccio)
+* [Roadmaps](https://github.com/verdaccio/verdaccio/projects)
+* [Reporting an issue](https://github.com/verdaccio/verdaccio/blob/master/CONTRIBUTING.md#reporting-a-bug)
 * [Running discussions](https://github.com/verdaccio/verdaccio/issues?q=is%3Aissue+is%3Aopen+label%3Adiscuss)
-* [Chat Room](http://chat.verdaccio.org/)
-* [Logos](https://github.com/verdaccio/verdaccio/tree/master/assets)
+* [Chat](http://chat.verdaccio.org/)
```

Comment:
This link seems to be broken, it says to make a new shorten link

Reply:
The chat one? It works for me.

Reply:
It looks broken on my phone but working on computer, so we can check later if it could be a problem (it says that the link is broken but offers a button with a correct redirect)

Reply:
It looks broken on my phone but working on computer, so we can check later if it could be a problem (it says that the link is broken but offers a button with a correct redirect)